src/components/Axis/Axis.jsx

```diff
@@ -0,0 +1,176 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { discreteTicks } from '../../util/chart-helpers';
+import { createClass } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-Axis');
+
+const {
+	array,
+	func,
+	number,
+	oneOf,
+	any,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["visualizations", "chart primitives"]}
+ *
+ * Axes and allies
+ *
+ * This component is a very close sister to `d3.avg.axis` and most of the logic
+ * was ported from d3.
+ */
+const Axis = createClass({
+	displayName: 'Axis',
+
+	statics: {
+		_lucidIsPrivate: true,
+	},
+
+	propTypes: {
+		/**
+		 * Classes are appended to root element along with existing classes using
+		 * the `classnames` library.
+		 */
+		className: any,
```

Comment:
Yeah, I flip flopped on this one. I feel like `any` is better since we're running stuff through `classnames` which could potentially be useful to a consumer.